feat: increase wizard image max height to 640px

fbricon opened the pull request:
Following up on https://github.com/Granite-Code/continue-for-granite/pull/54#issuecomment-2955248061

- increase wizard image max height to 640px
- when steps and images are stacked, image max height is set to 400px (as before)

https://github.com/user-attachments/assets/4558d29a-dcda-4341-871a-b011a26a1d71

We still need to crop the SVGs heights though

fbricon committed:
feat: increase wizard image height to 640px

Signed-off-by: Fred Bricon <fbricon@gmail.com>

fbricon commented:
@allanday do you want to merge this one as-is and then look at resizing the SVGs separately or...?

allanday commented:
@jimmac is it OK to simply crop the SVG in Inkscape, or is there more to it than that?

jimmac committed:
illustrations: fix canvas size

canvas now 600x500px. repositioned to remain relatively stable
between the steps.

jimmac commented:
After resizing the canvas, some repositioning was required to keep the boxes relatively stable between the steps as the bounding box changes. Can't push to fbricon's branch so here's the patch:

[0001-illustrations-fix-canvas-size.zip](https://github.com/user-attachments/files/20778454/0001-illustrations-fix-canvas-size.zip)

fbricon commented:
@jimmac thanks that's way better!

<img width="1480" alt="Screenshot 2025-06-18 at 11 48 16" src="https://github.com/user-attachments/assets/20638887-76df-4166-9bf3-ea457ebfd434" />

<img width="733" alt="Screenshot 2025-06-18 at 11 47 53" src="https://github.com/user-attachments/assets/c93a219f-8ec8-4912-b903-52120126e853" />

@jimmac FYI, when you can't push to a PR's branch directly, you can open a PR against that remote repo/branch. So I would merge your PR in my fork, which would automatically update the original PR.

allanday commented:
Those latest screenshots look good to me :+1:
